todo.py

```
@@ -1,13 +1,65 @@
 import json
+import os
+from datetime import datetime
+import numpy
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)